script-library/docs/sshd.md

```diff
@@ -21,7 +21,34 @@
 
 ## Usage
 
-This script can be used either ad-hoc in an already running container or in a Dockerfile.
+This script can be used either ad-hoc in an already running container or in a Dockerfile. 
+
+### Usage with a default container image 
```

Comment:
This repository is also used for Remote - Containers repository where no concept of a default image exists.  

What I'd recommend is we say, "Usage when this script is already in an image (e.g. the default Codespaces image - codespaces-linux)" or something like that.

Reply:
Got it.. that makes sense. I'll update the title and resubmit!